internal/util/copy.go

```diff
@@ -83,6 +99,45 @@ func CopyIndexFiles(logger *log.CmdLogger, src, dest string) error {
 	return nil
 }
 
+// skipFolder checks if the folder will need to be skipped during indexes copy.
+func skipFolder(logger *log.CmdLogger, indexSrcDir, src string) (bool, error) {
+	path, _ := filepath.Rel(indexSrcDir, src)
```

Comment:
Needs to handle any errors. I would also be tempted to do this outside the function and just pass in the path, although maybe the error handling would be neater if it stays here. Either would be fine.